ale/drivers/isis_spice_driver.py

```diff
@@ -0,0 +1,588 @@
+from glob import glob
+import os
+import struct
+import re
+
+import pvl
+import numpy as np
+import quaternion
+
+from ale import config
+from ale.drivers.base import Driver, Isis3
+
+def read_table_data(table_label, file):
+    """
+    Helper function to read all of the binary table data
+
+    Parameters
+    ----------
+    table_label : PVLModule
+        The ISIS table label
```

Comment:
I suspect that sphinx will complain that the `The ISIS...` is not aligned with `PVLModule`, but I would be very happy if it did not.